pkg/cloudprovider/registry/register.go

```diff
@@ -20,7 +20,7 @@ import (
 
 	"github.com/awslabs/karpenter/pkg/apis/provisioning/v1alpha4"
 	"github.com/awslabs/karpenter/pkg/cloudprovider"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
Why is this necessary?

Reply:
An epic battle with my linter -- which I lost. Will fix.